server/internal/handlers/comments.api.go

```diff
@@ -31,12 +31,31 @@ func GetReviewComments (w http.ResponseWriter, r *http.Request) {
 
 	comments, err := models.GetReviewComments(reviewId)
 
+
 	if err != nil {
 		log.Error(err)
 		api.InternalErrorHandler(w)
 		return
 	}
 
+
+	// Grab user Id from claims
```

Comment:
These are virtually identical pieces of code, maybe it would be good to have a util that
```go
loop stuffs
if stuff.UserId == claims.UserId
    stuff.OwnedBy = true
else
    stuff.OwnedBy = false
```